src/main/java/frc/robot/utils/NullableDouble.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.utils;
+
+import edu.wpi.first.util.struct.StructSerializable;
+
+/** Silly */
+public class NullableDouble implements StructSerializable {
+  final boolean isNull;
+  final Double value;
```

Comment:
@Lewis-Seiden Whole point of the class is to allow null Doubles to be serialized. You could certainly use a `double` and set it to an arbitrary value. Might as well just use the boxed version though.

@spellingcat It *does* mean we can get away without actually storing isNull though. It's just needed for serialization. You can do the same thing with NullableRotation2d now that I think about it.

Reply:
i think i made it a Double because in Module.java it checks if .get() == null

Reply:
Since it's returning null if the value is null, and otherwise the value, you can just return the value. is null is really only something that's needed in serialization. You can cut the variable here and just use it in the struct serialization implementation.